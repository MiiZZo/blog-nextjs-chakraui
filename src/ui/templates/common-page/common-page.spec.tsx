import React from "react";
import { cleanup } from "@testing-library/react";
import { myRender } from "@lib/testing/render";
import CommonPage from "./common-page";

afterEach(cleanup);

describe("common page component", () => {
  it("should match a shapshot", () => {
    const { asFragment } = myRender(
      <CommonPage footer={<div>footer</div>}>
        <div>children</div>
      </CommonPage>
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
