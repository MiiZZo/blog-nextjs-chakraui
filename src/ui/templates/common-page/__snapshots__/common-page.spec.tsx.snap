// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`common page component should match a shapshot 1`] = `
<DocumentFragment>
  <div
    class="css-o44is"
  >
    <div
      class="css-92qjs2"
    >
      <div>
        children
      </div>
    </div>
    <div>
      footer
    </div>
  </div>
</DocumentFragment>
`;
